lib/screens/wh/transfer/edit_fullscreen/edit_fullscreen.dart

```diff
@@ -261,7 +261,7 @@ class _LinesState extends State<Lines> {
     var schema = <Field>[
       // const Field(cBatch, StringType()),
       fGoods.copyWith(width: 3.0),
-      fUomAtQty.copyWith(width: 0.5, editable: false),
+      fUomAtQty.copyWith(width: 0.5),
```

Comment:
Can you remove this for now? because it has to stay read-only.

Reply:
Changed back